### PR TITLE
Provided Service Account support 

### PR DESCRIPTION
**Pull request by ahma**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no|yes
| Related tickets | #31 #187 
| License         | Apache 2.0

- [x] Test
- [x] Documentation

### CODE REVIEW

File: charts/nginx-logging-demo/README.md [hunk 1]
```diff
@@ -40,29 +40,34 @@ The command removes all the Kubernetes components associated with the chart and
...
-| `loggingOperator.fluentd.metrics.timeout`                 | Scrape timeout.                                         | ``             |
-| `loggingOperator.fluentd.metrics.interval`                | Scrape interval.                                        | ``             |
-| `loggingOperator.fluentbit.metrics.serviceMonitor`        | Enable to create ServiceMonitor for Prometheus operator | `false`        |
-| `loggingOperator.fluentbit.metrics.prometheusAnnotations` | Add prometheus labes to fluent pods.                    | `false`        |
-| `loggingOperator.fluentbit.metrics.port`                  | Metrics Port.                                           | ``             |
-| `loggingOperator.fluentbit.metrics.path`                  | Metrics Path                                            | ``             |
-| `loggingOperator.fluentbit.metrics.timeout`               | Scrape timeout.                                         | ``             |
-| `loggingOperator.fluentbit.metrics.interval`              | Scrape interval.                                        | ``             |
```
Comment (baluchicken) [inline charts/nginx-logging-demo/README.md]: Do you really want to remove `loggingOperator.fluentbit.metrics.interval` ?

File: pkg/resources/fluentbit/rbac.go [hunk 2]
```diff
@@ -23,41 +23,71 @@ import (
...
-		},
-	}, k8sutil.StatePresent
+			r.Logging.QualifiedNamespacedName(clusterRoleBindingName),
+			r.Logging.Labels,
+			r.Logging,
+		),
+		RoleRef: rbacv1.RoleRef{}}, k8sutil.StateAbsent
 }
 
 func (r *Reconciler) serviceAccount() (runtime.Object, k8sutil.DesiredState) {
```
Comment (baluchicken) [inline pkg/resources/fluentbit/rbac.go]: Shouldn't we use the getServiceAccount() method here?

File: pkg/resources/fluentd/rbac.go [hunk 3]
```diff
@@ -22,39 +22,71 @@ import (
...
+	}
+	return &rbacv1.RoleBinding{
+		ObjectMeta: templates.FluentdObjectMeta(
+			r.Logging.QualifiedName(roleBindingName),
+			r.Logging.Labels, r.Logging,
+		),
+		RoleRef: rbacv1.RoleRef{}}, k8sutil.StateAbsent
 }
 
 func (r *Reconciler) serviceAccount() (runtime.Object, k8sutil.DesiredState) {
```
Comment (baluchicken) [inline pkg/resources/fluentd/rbac.go]: Shouldn't we use the getServiceAccount() method here?

File: charts/nginx-logging-demo/README.md (hunk 1 above)
Comment (ahma) [reply charts/nginx-logging-demo/README.md]: No you're right. thanks

**Pull request merged**